### PR TITLE
Add first Test and first Job for RevPi_Core in LAVA Server [REVPI-2538]

### PR DESCRIPTION
**Pull request by RamiGspo**
Add Basic Test to be run from repository.
Add first job for GitHub repository.

### CODE REVIEW

**Comment by zhan-kunbus** [comment]
The Signed-Off-By is missing in commit messages

File: automated/linux/jobs-definition/job-0001_test-repo.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    timeout:
+      minutes: 2
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository: https://github.com/RevolutionPi/LAVA-test-definitions.git
+      from: git
+      path: automated/linux/basic-tests/basic-test-0001_repo.yaml
+      name: basic-test-0001_repo
```
Comment (zhan-kunbus) [inline automated/linux/jobs-definition/job-0001_test-repo.yaml]: if it works, I would suggest a "\n" here for every file to make it no warning from git

File: automated/linux/jobs-definition/job-0001_test-repo.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+visibility: public
+
+timeouts:
+  job:
+    minutes: 5
+  action:
+    minutes: 1
+  connection:
+    minutes: 2
+
```
Comment (zhan-kunbus) [inline automated/linux/jobs-definition/job-0001_test-repo.yaml]: there are some not needed spaces here

File: automated/linux/basic-tests/basic-test-0001_repo.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    scope:
+        - functional
+    devices:
+        - Core3_Ostfildern_001
+
+run:
+    steps:
+        - lava-test-case test1 --result pass
+        - lava-test-case test2 --result pass
+#        - lava-test-case test3 --result fail
```
Comment (zhan-kunbus) [inline automated/linux/basic-tests/basic-test-0001_repo.yaml:18]: if these two are not needed, maybe they should be removed.

Comment (RamiGspo) [reply automated/linux/basic-tests/basic-test-0001_repo.yaml:18]: They are not necessary.
It simply shows how to define a failed test.

**Comment by zhan-kunbus** [comment]
the commit "Add description... " might should be "squash" to previous commits.

File: automated/linux/basic-tests/basic-test-0001_repo.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        - raspian
+
+    scope:
+        - functional
+    devices:
+        - Core3_Ostfildern_001
+
+run:
+    steps:
+        - lava-test-case test1 --result pass
```
Comment (zhan-kunbus) [inline automated/linux/basic-tests/basic-test-0001_repo.yaml:15]: Wo/was ist die "test1" ?

Comment (RamiGspo) [reply automated/linux/basic-tests/basic-test-0001_repo.yaml:15]: That is just a name... It can be called differently.

Comment (zhan-kunbus) [reply automated/linux/basic-tests/basic-test-0001_repo.yaml:15]: so, this means no responding testcase in test script defined.
A suggestion would be,  maybe make test case with python, a testcase with shell, and perl, etc, empty test case.  and then call them in YAML. Both to test the environment  also make examples for every kind of script.

**Pull request closed without merging**